packages/server/src/stages/route-import-manifest/route-import-manifest.ts

```diff
@@ -149,9 +149,19 @@ export function parseParametersFromRoute(
  * We're not fooled by you, `yarn workspace`!
  */
 function findNodeModulesRoot(src: string) {
-  const nodeModules = join(src, "node_modules")
-  const includesBlitzPackage = fs.existsSync(join(nodeModules, "blitz"))
-  return includesBlitzPackage ? nodeModules : join(nodeModules, "../../../node_modules")
+  let nodeModules = join(src, "node_modules")
+  let includesBlitzPackage = fs.existsSync(join(nodeModules, "blitz"))
+  let count = 0
+  while (!includesBlitzPackage) {
+    // Check for node_modules at the next level up
+    nodeModules = join(nodeModules, "../../node_modules")
+    includesBlitzPackage = fs.existsSync(join(nodeModules, "blitz"))
+    count++
+    if (count > 5) {
+      throw new Error("Oops. We can't find the blitz node_modules package. Are you sure it's installed?")
```

Comment:
@kenkuan So the test in `packages/server` are failing (run `yarn test` inside `packages/server` to see). I think instead of throwing an error here, we should log a warning and default to `<src>/node_modules`. This will make tests pass and have a fallback like before.

Reply:
@flybayer Cool. I made the changes in 65880b1.